docker-dev.sh

```diff
@@ -0,0 +1,8 @@
+# Copies the current directory into the Docker container as a volume and runs Jekyll
+# inside the container on port 9001.
+MSYS_NO_PATHCONV=1 docker run --rm \
+  --name design-system-site \
+  -v "$PWD":/usr/src/app \
+  -p 9001:9001 \
+  -it starefossen/github-pages \
```

Comment:
@ediblecode To get the process to run I need to remove --interactive and --tty. If I leave either, I get the error `the input device is not a TTY. If you are using mintty, try prefixing the command with 'winpty'`

Reply:
That's not to say it doesn't run - you just don't get an interactive session by default. You can always attach after the fact.